GoogleSignIn/Sources/GIDKeychainHandler/GIDKeychainHandler.h

```diff
@@ -20,7 +20,7 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-@protocol GIDKeychainHandlling <NSObject>
+@protocol GIDKeychainHandler <NSObject>
```

Comment:
If you have an `Implementation` directory for implementations, then you could put protocols under an `API` or `Interface` directory. Up to you.